tera-templates/releases/build_queue.html

```diff
@@ -0,0 +1,126 @@
+{%- extends "base.html" -%}
+{%- import "releases/header.html" as release_macros -%}
+
+{%- block title -%}Build Queue - Docs.rs{%- endblock title -%}
+
+{%- block header -%}
+    {{ release_macros::header(title="Build Queue", description=description, tab="queue") }}
+{%- endblock header -%}
+
+{%- block body -%}
+    <div class="container">
+        <div class="recent-releases-container">
+
+            <div class="release">
+                {% set queue_length = queue | length -%}
+                {%- if queue_length == 0  -%}
+                    <strong>There is nothing in queue</strong>
+                {%- else -%}
+                    <strong>Queue</strong>
+                {%- endif %}
+            </div>
+
+            <ol class="queue-list">
+                {% for crate in queue -%}
+                    <li>
+                        <a href="https://crates.io/crates/{{ crate.name }}">
+                            {{ crate.name }} {{ crate.version }}
+                        </a>
+
+                        {% if crate.priority != 0 -%}
+                            (priority: {{ crate.priority }})
+                        {%- endif %}
+                    </li>
+                {%- endfor %}
+            </ol>
+        </div>
+    </div>
+{%- endblock body -%}
+
+{%- block javascript -%}
+    <script type="text/javascript" charset="utf-8">
+        function getKey(ev) {
+            if ("key" in ev && typeof ev.key != "undefined") {
+                return ev.key;
+            }
+            return String.fromCharCode(ev.charCode || ev.keyCode);
+        }
+
+        document.getElementById("i-am-feeling-lucky-button").onclick = function () {
+            var form = document.getElementsByClassName("landing-search-form");
+            var input = document.createElement('input');
+            input.type = 'hidden';
+            input.name = 'i-am-feeling-lucky';
+            input.value = 1;
+            document.getElementsByClassName("landing-search-form")[0].appendChild(input);
+            return true;
+        };
+
+        function handleShortcut(ev) {
+            if (ev.ctrlKey || ev.altKey || ev.metaKey || document.activeElement.tagName === "INPUT") {
+                return;
+            }
+            switch (getKey(ev)) {
+                case "s":
+                case "S":
+                    ev.preventDefault();
+                    document.getElementById("search").focus();
+                    break;
+            }
+        }
```

Comment:
This looks copy/pasted. Please refactor into a separate template (but see also the comment on the first time this showed up).